modules/client-configuration/pages/clients-sle.adoc

```diff
@@ -53,7 +81,7 @@ To add the client tools, add these channels from the command prompt:
 
 |===
 
-include::snippets/add_channels_cli.adoc[]
+include::snippets/add_channels_scc.adoc[]
```

Comment:
This is for Uyuni, this is for openSUSE, but the channel names are wrong, and we don't use SCC for openSUSE on Uyuni.

Reply:
This is for client tools on SLE under Uyuni:

![image](https://user-images.githubusercontent.com/3914967/98321676-6c749c80-2031-11eb-8120-94a769bf2008.png)

openSUSE under Uyuni is in the openSUSE chapter.

Reply:
Not for this PR I guess but: Why do we use different structures? Shouldn't we then have openSUSE on a separate chapter for SUSE Manager as well?

As you can see it makes the reviews confusing, and can confuse also people changing from Uyuni to SUSE Manager.

Reply:
openSUSE is its own chapter, it contains instructions for both SUMA and Uyuni.